test/rebar3_sbom_json_SUITE.erl

```diff
@@ -486,3 +512,22 @@ check_licenses_constraints(Licenses) ->
     lists:foreach(fun(License) ->
         ?assertMatch(#{<<"license">> := #{<<"id">> := _}}, License)
     end, Licenses).
+
+check_cpe_format(Cpe) ->
+    ?assertNotEqual(nomatch, re:run(Cpe, ?CPE_REGEX)).
+
+check_cpe_content(#{<<"name">> := <<"hex_core">>} = Component) ->
+    #{<<"version">> := Version, <<"cpe">> := Cpe} = Component,
+    check_cpe_format(Cpe),
+    ?assertEqual(<<"cpe:", ?CPE_VERSION/bitstring, ":a:hex:hex_core:",
+                   Version/bitstring, ":*:*:*:*:*:*:*">>, Cpe);
+check_cpe_content(Component) ->
```

Comment:
To check if the link parsing works we could check for the organisation of on of the example dependencies here like maps or grisp.